doc/.wordlist.txt

```diff
@@ -297,6 +298,7 @@ VMs
 VPD
 VPN
 VPS
+vsock
```

Comment:
vsock is not a word - and you probably don't need it in here anyway (I didn't see the failure, but it was probably because of wrong indentation?).

Reply:
Right, it shows in the text but with `` around it so probably OK. Thanks!